Makefile

```diff
@@ -19,6 +19,22 @@ getdeps:
 	@echo "Installing misspell" && go get -u github.com/client9/misspell/cmd/misspell
 	@echo "Installing ineffassign" && go get -u github.com/gordonklaus/ineffassign
 
+prunevendor:
+	@find vendor/ -name '.editorconfig' | xargs -I{} rm -v {}
```

Comment:
Add this to CONTRIBUTING.md as well to deprecate `pkg-add`, `pkg-remove` etc. remove them from Makefile. We need to update the CONTRIBUTING.md respectively.

Reply:
Will do.